README.md

```diff
@@ -53,6 +63,11 @@ Run the following command to initialize your project:
 ```bash
 $ ./helpers/init_project.sh
 ```
+> Note: If you run into an error message that looks like `env: bash\r: No such file or directory`, this means that you've configured GIT to use Windows line endings, which cannot be interpreted properly by Linux terminals. To fix this, you can execute
```

Comment:
Not sure if projects that are cloned directly in WSL suffer from this issue, as they will be cloned from the linux version of git.